pkgs/applications/version-management/radicle-cli/default.nix

```diff
@@ -2,86 +2,38 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, pkg-config
-, cmake
-, installShellFiles
-, asciidoctor
 , DarwinTools
-, openssl
-, libusb1
 , AppKit
-, git
-, openssh
-, testers
-, radicle-cli
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "radicle-dev";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LS6zYpMg0LanRL2M8ioGG8Ys07TPT/3hP7geEGehwxg=";
+    repo = "heartwood";
+    rev = "b27313bc5c703c8a25343176c620c8652b815abc";
+    sha256 = "sha256-WyLqTOF1TOuRD4/aTikeOR446fd00opryiUnAdPfA9Q=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "automerge-0.0.2" = "sha256-MZ1/rca8ZsEUhd3bhd502PHlBbvqAOtnWFEdp7XWmYE=";
-      "automerge-0.1.0" = "sha256-dwbmx3W13oZ1O0Uw3/D5Z0ht1BO1PmVVoWc/tLCm0/4=";
-      "cob-0.1.0" = "sha256-ewPJEx7OSr8X6e5QJ4dh2SbzZ2TDa8G4zBR5euBbABo=";
-      "libusb1-sys-0.6.2" = "sha256-577ld1xqJkHp2bqALNq5IuZivD8y+VO8vNy9Y+hfq6c=";
-      "walletconnect-0.1.0" = "sha256-fdgdhotTYBmWbR4r0OMplOwhYq1C7jkuOdhKASjH+Fs=";
-    };
-  };
-
-  # Otherwise, there are errors due to the `abigen` macro from `ethers`.
-  auditable = false;
+  cargoHash = "sha256-ZKIGaHTIvlkVIPhuGOYTG0zXBOZoNxtcNIVZJyuNa+8=";
 
-  nativeBuildInputs = [
-    pkg-config
-    cmake
-    installShellFiles
-    asciidoctor
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+  nativeBuildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     DarwinTools
   ];
 
-  buildInputs = [
-    openssl
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    libusb1
+  buildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     AppKit
   ];
 
-  postInstall = ''
-    for f in $(find . -name '*.adoc'); do
-      mf=''${f%.*}
-      asciidoctor --doctype manpage --backend manpage $f -o $mf
-      installManPage $mf
-    done
-  '';
-
-  nativeCheckInputs = [
-    git
-    openssh
-  ];
-  preCheck = ''
-    eval $(ssh-agent)
-  '';
-
-  passthru.tests = {
-    version = testers.testVersion { package = radicle-cli; };
-  };
+  doCheck = false;
 
   meta = {
     description = "Command-line tooling for Radicle, a decentralized code collaboration network";
     homepage = "https://radicle.xyz";
-    license = lib.licenses.gpl3Plus;
-    maintainers = with lib.maintainers; [ amesgen ];
+    license = lib.licenses.mit;
```

Comment:
good point

https://app.radicle.xyz/seeds/seed.radicle.xyz/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5 should be the source of truth however ;)

Reply:
@gsaslis hehe I also told @Luz that but I think for now it's not that bad to follow the binaries that GitHub CI outputs currently.
Once we have CI on Radicle we should change that for sure and use `fetchGit` instead of `fetchFromGitHub`

Reply:
If i understand this correctly, we could do two steps:
1) Change to `fetchGit` and use the git repo on radicle's server: https://seed.radicle.xyz/z3gqcJUoA1n9HaHKufZs5FCSGazv5.git
2) Change to `fetchRadicle` (which probably doesnt exist yet) and use the source of truth according gsaslis
Should I do "1)"?

Reply:
I think if we as next step, maybe once Radicle moves CI to it's own solution, move to `fetchGit` that should already be enough.
I don't think we need a `fetchRadicle` fn since Radicle is built on top of git.